src/main/webapp/app/account/index.ts

```diff
@@ -18,4 +19,4 @@ export * from './settings/settings.component';
 export * from './settings/settings.route';
 export * from './social/social-register.component';
 export * from './social/social.route';
-export * from './account.route';
+export * from './account.route';
```

Comment:
Hey,
What has changed between Line 21 and Line 22?

Reply:
apparently a new line character is removed... shouldn't affect anything :)